Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN pipenv requirements --dev > requirements-dev-lock.txt
 
 FROM amazon/aws-lambda-python:${PYTHON} AS zip
 RUN yum install -y git zip
-COPY src/sync.py .
+COPY src .
```

Comment:
This is why it wasn't able to find the events modules

Reply:
Ah, nice catch!